### ISSUE TITLE
[🐛 Bug]: have a switch to allow offline mode, and don’t check chrome labs when not using chrome

### ISSUE THREAD

**Issue by step21** [webdriverio/webdriverio#12705]
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md#reporting-new-issues).

### WebdriverIO Version

Latest

### Node.js Version

20

### Mode

Standalone Mode

### Which capabilities are you using?

```typescript
capabilities: [{
        maxInstances: 5,
        browserName: 'firefox',
        acceptInsecureCerts: true,
        'moz:firefoxOptions': {
            args: ['-headless']
        }
    }],
```

### What happened?

I expected to be able to run my tests locally, while being offline due to being on the train or other reasons. Additionally, why is chrome labs being checked when only using Firefox?

### What is your expected behavior?

I expected the test to run if it has all the necessary browsers available. It is not an option for me to harcode as mentioned here https://github.com/webdriverio/webdriverio/issues/10965 as I only need it when offline and do not have custom images. Also imo Firefox should not require a call to chrome labs.

### How to reproduce the bug.

Just turn off your internet, have only Firefox in capabilities. Run any test. It will still check chromelabs and not work. 

### Relevant log output

```typescript
Execution of 1 workers started at 2024-04-16T14:01:00.207Z

Error: Error: getaddrinfo ENOTFOUND googlechromelabs.github.io
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:107:26)
Error: getaddrinfo ENOTFOUND googlechromelabs.github.io
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:107:26) {
  errno: -3008,
  code: 'ENOTFOUND',
  syscall: 'getaddrinfo',
  hostname: 'googlechromelabs.github.io'
}
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

**Comment by christian-bromann** [comment]
> I expected to be able to run my tests locally, while being offline due to being on the train or other reasons.

As per our documentation this works if:

> If you omit browserVersion, WebdriverIO will first try to locate and use an existing installation with [locate-app](https://www.npmjs.com/package/locate-app), otherwise it will download and install the current stable browser release.

So I assume that this application look up has failed which caused WebdriverIO to try to download the binary and driver.

> Additionally, why is chrome labs being checked when only using Firefox?

I am not sure why you are experiencing this, I can't reproduce it on my side as I am getting `Error: getaddrinfo ENOTFOUND product-details.mozilla.org`

Do you have Firefox installed on your OS? If so which path is it on? What OS are you using?

**Comment by step21** [comment]
Thanks for the quick reply. I copied a more readable error message now, somehow it copied weirdly formatted on my phone.
I definitely did not specify a browser version. (should I paste my wdio.conf.js somewhere?)
This is on macOS, and I use Firefox everyday. Also when I am online, it doesn't seem to always redownload.
I just ran `locate-app` manually, and that gave `'/Applications/Firefox.app/Contents/MacOS/Firefox'` as a path. This was on node v18 (in my test environment I use both)
Does any of that help?

**Comment by step21** [comment]
> @step21 thanks for further details, it seems that WebdriverIO unnecessarily tries to resolve the Firefox version, even though it has all required information to run the test. This causes an unnecessary network hop. A possible fix would be 
> ...
> Feel free to raise a PR with the change set.

Oh wow thanks, that is very helpful, will try to make a PR.

**Comment by christian-bromann** [comment]
This bug was resolved in #12718
